ee/desktop/menu/menu_template.go

```diff
@@ -4,6 +4,14 @@ import (
 	"fmt"
 	"strings"
 	"text/template"
+	"time"
+
+	"github.com/xeonx/timeago"
```

Comment:
My inclination would be to write our own, with rules from the K2 side, over pulling in a library. Do you think this is worth pulling in?

Reply:
Nah, it does more than we need atm (like `i18n` support). I'll replace with something simple to get us going.